src/django_pg_migration_tools/operations.py

```diff
@@ -271,8 +271,11 @@ def safer_drop_index(
     def _set_lock_timeout(
         self, schema_editor: base_schema.BaseDatabaseSchemaEditor, value: str
     ) -> None:
-        cursor = schema_editor.connection.cursor()
-        cursor.execute(TimeoutQueries.SET_LOCK_TIMEOUT, {"lock_timeout": value})
+        schema_editor.execute(
+            psycopg_sql.SQL(TimeoutQueries.SET_LOCK_TIMEOUT)
+            .format(lock_timeout=psycopg_sql.Literal(value))
+            .as_string(schema_editor.connection.connection)
+        )
```

Comment:
🍷 `schema_editor.execute` takes a string. I had to transform that into a proper `SQL` composable object to avoid using f-strings with direct sql statements.